### ISSUE TITLE
Style attribute with a map as value: doesn't evaluate code within the map

### ISSUE THREAD

**Issue by luontola** [weavejester/hiccup#211]
Input:
```clj
[:p {:style {:color (if true
                      "red"
                      "blue")}}
 "hello"]
```

Expected output:
```html
<p style="color:red;">hello</p>
```

Actual output:
```html
<p style="color:(if true &quot;red&quot; &quot;blue&quot;);">hello</p>
```

Hiccup version: 2.0.0-RC3

**Comment by teodorlu** [comment]
I believe I'm hitting the same issue.

Unexpected behavior: `theme` doesn't evaluate.

```clojure
(let [theme {:theme/primary-color "black"}]
  (str (hiccup2.core/html [:div {:style {:css.prop/color (:theme/primary-color theme)
                                         :css.prop/font-size "1.2rem"}}
                           "Write your weeknote:"])))
;; => "<div style=\"color:(:theme/primary-color theme);font-size:1.2rem;\">Write your weeknote:</div>"
```

Workaround: wrap style in `identity`:

```clojure
(let [theme {:theme/primary-color "black"}]
  (str (hiccup2.core/html [:div {:style (identity {:css.prop/color (:theme/primary-color theme)})}
                           "Write your weeknote:"])))
;; => "<div style=\"color:black;\">Write your weeknote:</div>"
```

**Comment by luontola** [comment]
I think this has been fixed by #214. @weavejester May we have a new release, please?

**Comment by weavejester** [comment]
Fix released. Apologies for the delay.
